api/lib/domain/usecases/find-paginated-saved-tutorials.js

```diff
@@ -1,7 +1,15 @@
-module.exports = async function findSavedTutorials({ tutorialEvaluationRepository, tutorialRepository, userId } = {}) {
+const paginateModule = require('../../infrastructure/utils/paginate');
```

Comment:
```suggestion
const { paginate } = require('../../infrastructure/utils/paginate');
```
nitpick: Pour supprimer le paginateModule appelé ligne 12

Reply:
Alors ça ne fonctionne pas pour des soucis de stub ensuite avec sinon (cf: https://stackoverflow.com/a/52591287).